start.js

```diff
@@ -329,8 +329,7 @@ Promise.all([dnsServerStarted, httpServerStarted]).then(() => {
 
 		// Render everything
 		screen.render();
-		let ipAddr = argv.ip || ip.address();
-		repl.write("Waiting for DNS connection at ip " + ipAddr + "...");
+		repl.write("Waiting for DNS connection at ip " + (argv.host || ip.address()) + "...");
```

Comment:
We're waiting for dns requests (not connections) on `argv.host`, but the important part is that we're responding with the ip in `ipAddr`. `ipAddr` had already been defined on line 62. I don't know if it's still in scope here or not, but if it's not, you can probably move the declaration to bring it in scope here.